ft_regressconfound.m

```diff
@@ -88,15 +93,14 @@
 datain = ft_checkdata(datain, 'datatype', {'timelock', 'freq', 'source'}, 'feedback', 'yes');
 
 % ensure that the required options are present
-cfg = ft_checkconfig(cfg, 'required', {'confound'}, 'renamed', {'Ftest','ftest'});
+cfg = ft_checkconfig(cfg, 'required', {'confound'}, 'renamed', {'Ftest','ftest'}, 'deprecated', {'beta','model'});
```

Comment:
since cfg.beta and cfg.model are not supported any more, they should be flagged as forbidden rather than deprecated. 

They would be deprecated if you would have added code for backward compatibility, mapping the old options onto the new one.